extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-alert.js

```diff
@@ -0,0 +1,178 @@
+import {LitElement, html, css} from 'lit';
```

Comment:
I like this component. I have started UI related component under the qui namespace, so the live in `extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qui/` and then rather than qwc- it starts with qui-.  (see https://github.com/quarkusio/quarkus/pull/31441/files#diff-b22eff93cdc1dfd4bae6663a9de45fc724c050e83f7bbe191bdf1bb1524f4468) Can you move this UI component to there and rename ? You can also add it to the BuildTimeContentProcessor (see https://github.com/quarkusio/quarkus/pull/31441/files#diff-4f2056639905c495c56b6a5bc3ba7b3b9f2927d19b0032d0015c94b2bb378a86) so that it can be imported easily.